enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/WebSocketAPI.java

```diff
@@ -160,6 +160,10 @@ public ResponseObject process(RequestContext requestContext) {
         ResponseObject responseObject = new ResponseObject();
         responseObject.setRequestPath(requestContext.getRequestPath());
         responseObject.setApiUuid(apiConfig.getUuid());
+        // TODO (thushani) manage resource level security disabling
+        if (apiConfig.isDisableSecurity()) {
```

Comment:
Here it's only checking for the API level, we have to do it for each channel level.

Reply:
We can move `buildResource` function in the choreo branch to a util class.

https://github.com/wso2/product-microgateway/blob/f220ef11fd3e177a10f397d3bcc8b8f84b481137/enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/RestAPI.java#L221

As we've done in the main branch

https://github.com/wso2/product-microgateway/blob/d24eb4d6e389daf2851d9a534f51dcf27f9c07db/enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/Utils.java#L87